### DISCUSSION TITLE
Lotus Node Earnings

### DISCUSSION THREAD

**Discussion by KB-DIL** [filecoin-project/community#250] [Mining]
I recently used AWS to deploy a Lotus node. The system status has been active for the past few days but the node has yet to earn any FIL. I see that I can enable lotus-miner, but I was under the impression that the lotus node as is should be able to earn rewards. Is there something I'm doing wrong? At what rate do these nodes accrue FIL tokens?

**Comment by rjan90** [comment]
Hey! A lotus node only keeps track of the chain, and does not win any block rewards! To be a storage provider on the Filecoin network you need to run the lotus-miner, and pledge both FIL and storage in a computational heavy process that will take time 😄 Only after pledging over 10TiB you’ll be eligable for block rewards. I would reccomend going through the docs here: https://docs.filecoin.io/about-filecoin/how-filecoin-works/

**Reply by KB-DIL** [reply]
Got it - thank you!

**Reply by KB-DIL** [reply]
@rjan90 I'm having trouble finding the minimum FIL required for mining. I've reviewed the hardware requirements which are feasible, but what is the required FIL pledge?

**Reply by rjan90** [reply]
You can see the sector-pledge per TiB here: https://fgas.io/
